app/konnect/runtime-manager/kic.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Kong Ingress Controller Integration
+content_type: explanation
+beta: true
+---
+
+You can use native Kubernetes resources to configure your clusters in {{site.konnect_short_name}} by integrating your Kong Ingress Controller (KIC) deployment with {{site.konnect_short_name}}. 
+This setup is ideal for organizations who want to manage gateways in {{site.konnect_short_name}} through native Kubernetes resources without having to use a hybrid deployment model. 
+
+## About KIC in {{site.konnect_short_name}}
+
+{{site.kic_product_name}} (KIC) for Kubernetes configures {{site.base_gateway}} using Ingress or [Gateway API](https://gateway-api.sigs.k8s.io/) resources created inside a Kubernetes cluster. 
+
+Beyond proxying the traffic coming into a Kubernetes cluster, KIC also lets you configure plugins, load balancing, health checking, and leverage all that {{site.base_gateway}} offers in a standalone installation. For more information about KIC, see [{{site.kic_product_name}} Design](/kubernetes-ingress-controller/latest/concepts/design/). 
+
+By integrating your KIC deployment with {{site.konnect_short_name}}, this read-only integration allows you to view your runtime entities, such as routes and applications, from your Kubernetes resources in {{site.konnect_short_name}}.  
+
+Here are a few benefits of running KIC in {{site.konnect_short_name}} over a self-managed setup:
+* **Easy to set up:** The wizard allows you to add your KIC runtime group to {{site.konnect_short_name}} in minutes.
+* **Monitor your KIC analytics:** By integrating with {{site.konnect_short_name}}, you can view the analytics from your KIC runtime instances alongside any of your self-managed {{site.konnect_short_name}} runtime instances. 
+* **Display KIC entities in Dev Portal:** Publish your KIC services to the Dev Portal and make the API specs available to third-party developers. 
```

Comment:
Dev Portal integration has not been released yet for KIC. Should we keep this benefit in for now? Or mark it as "future"capabilities?